src/wallet/wallet.cpp

```diff
@@ -2784,46 +2784,6 @@ CAmount CWallet::GetNormalizedAnonymizedBalance() const
     return nTotal;
 }
 
-// Calculate total balance in a different way from GetBalance. The biggest
```

Comment:
15596: You should also be doing the pcoin -> wtx here where possible no? At least one example is https://github.com/dashpay/dash/blob/06cc4bbe5f1364800a66b31c61a4794f3fe90128/src/wallet/wallet.cpp#L4365

Reply:
I've attempted to use https://github.com/bitcoin/bitcoin/pull/15596/commits/faa3a246e8809b4954a4a6d202fe8c7f7f776b6e and it breaks builds for some reason